build.sbt

```diff
@@ -151,7 +151,7 @@ lazy val scapegoatSettings = Seq(
 
 lazy val mimaSettings = Seq(
   // First 2.13 release of 1.x
```

Comment:
this comment probably doesn't apply anymore then?

Reply:
yes indeed